### PR TITLE
Set Paywalls Tester deployment target to iOS 15

### PR DESCRIPTION
**Pull request by jamesrb1**
Updates the target of Paywalls Tester to iOS 15 so that we can test RevenueCatUI easily.

### CODE REVIEW

**Comment by jamesrb1** [comment]
@RCGitBot please test

**Comment by jamesrb1** [comment]
@RCGitBot please test

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift [hunk 1]
```diff
@@ -21,8 +21,10 @@ struct LoginWall<ContentView: View>: View {
         case .unknown:
             ProgressView()
                 .displayError(self.$error)
-                .task { @MainActor in
-                    await reload()
+                .onAppear {
+                    Task {
+                        await reload()
+                    }
```
Comment (jamesrb1) [inline Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift:27]: For some reason the .task caused an error (and subsequent error dialog) to pop up when run on iOS 15 (worked as expected on 16 and 17).

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/CompatibilityContentUnavailableView.swift
Comment (jamesrb1) [inline Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/CompatibilityContentUnavailableView.swift:1]: Taken from Customer Center, which I assume will remain internal so unavailable to access from Paywalls Tester.

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift (hunk 1 above)
Comment (joshdholtz) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift:27]: Can you leave a comment in there for this? 😅  This is something that one of us (probably me) would change in the future and not realize why we did this 😛

**Review by joshdholtz — APPROVED** [review]
This looks good to me! Just one small nit about moving something to a comment in the code but other than that let's :shipit:

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift (hunk 1 above)
Comment (jamesrb1) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift:27]: Good idea, will do!

**Review by JayShortway — APPROVED** [review]
Nice! I ran into the iOS 15 incompatibility a while back. Thanks for taking care of it!

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift (hunk 1 above)
Comment (jamesrb1) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/Login/LoginWall.swift]: Done!

**Comment by jamesrb1** [comment]
@RCGitBot please test

**Pull request merged**